### PR TITLE
feat(misc): update shared ts-node and ts-loader packages

### PR DESCRIPTION
**Pull request by jaysoo**
This PR updates `ts-loader` and `ts-node` that includes fixes and performance improvements (`ts-loader` now defaults to `transpileOnly: true` if the type checker plugin is used.

## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by barbados-clemens — APPROVED** [review]
all good from the Cypress/Jest side 👍

File: packages/jest/migrations.json [hunk 1]
```diff
@@ -136,6 +136,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "14.5.3": {
+      "version": "14.5.3-beta.0",
+      "packages": {
+        "jest": {
```
Comment (leosvelperez) [inline packages/jest/migrations.json:143]: I guess this should be `ts-node`, isn't it?

**Review by leosvelperez — CHANGES_REQUESTED** [review]
We also need to change the version here https://github.com/nrwl/nx/blob/master/packages/angular/src/utils/versions.ts#L19.

File: packages/jest/migrations.json (hunk 1 above)
Comment (barbados-clemens) [reply packages/jest/migrations.json:143]: yes, this should be ts-node. Thanks for catching!

**Pull request merged**